hummingbird/ml/_topology.py

```diff
@@ -208,11 +210,6 @@ def convert(topology, backend, test_input, device, extra_config={}):
             )
 
         if backend == onnx.__name__:
-            # vers = LooseVersion(torch.__version__)
-            # allowed_min = LooseVersion("1.6.0")
-            # Pytorch <= 1.6.0 has a bug with exporting GEMM into ONNX.
-            # For the moment only tree_trav is enabled for pytorch <= 1.6.0
-            # if vers < allowed_min:
             extra_config[constants.TREE_IMPLEMENTATION] = "tree_trav"
```

Comment:
Can we please leave this comment. Eventually when we will move to PyTorch > 1.6 we could remove the condition (assuming they fixed the bug)

Reply:
Ok, to clarify you mean leave the comment but still remove the condition?

Reply:
I think we can leave this as is right now. In the future we may want to remove the condition (as well as the comment). Like if the minimum version of pytorch we support will become 1.7 and onnx works.